android/lib/build.gradle

```diff
@@ -15,7 +15,7 @@ android {
 }
 
 dependencies {
-  compile 'com.facebook.react:react-native:+'
+  compile 'com.facebook.react:react-native:0.29.0'
   compile "com.google.android.gms:play-services-base:8.4.0"
```

Comment:
Nothing would build if the dependency isn't there. RN APIs are used all over the place in this project